src/graphql/data/TopTokens.ts

```diff
@@ -167,7 +167,8 @@ interface UseTopTokensReturnValue {
   tokens: TopToken[] | undefined
   hasMore: boolean
   loadMoreTokens: () => void
-  maxFetchable: number
+  tokenListLength: number
+  loadingRowsCount: number
```

Comment:
Can you omit `loadingRowsCount` and just use `tokenListLength`?

Reply:
I think splitting it into two vars adds unnecessary complexity.

Reply:
can't, see line 130 on TokenTable. there would be information loss